notebooks/02 - Custom layers.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "71c77df9-1d42-4993-9a0a-d311064d03c9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from unfolded.map_sdk import UnfoldedMap\n",
+    "unfolded_map = UnfoldedMap(\n",
+    "   # mapUrl='http://localhost:8080/incognito'\n",
+    "   # mapUrl='https://staging.unfolded.ai/incognito'\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "67fbd3d3-2eb6-45f4-b3e4-7391fc5282a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sidecar import Sidecar\n",
+    "sc = Sidecar(title='Unfolded map demo', anchor='split-right')\n",
+    "with sc:\n",
+    "    display(unfolded_map)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "05b6f62b",
+   "metadata": {},
+   "source": [
+    "## Load data into a DataFrame "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3366a68b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uuid import uuid4\n",
+    "dataset_id=uuid4()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "37ebfcb5",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<Future pending>"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "unfolded_map.add_dataset({\n",
```

Comment:
I'm thinking does it make sense to add text above these codeblocks that says something like:
`Using Map SDK's add_dataset function (we link it to the docs) you are able to add...`